app/assets/images/sephora_style_guide/svgs/black-tier.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These svgs seems a little heavy (e.g. unnecessary desc, usage or dimensions + viewbox, etc). Anyway we can simplify this?

Reply:
Removed few bits but dimensions + viewbox is pretty much on all svgs.

Reply:
You can just pass it through svgomg that will process and optimize it fully.